lib/infrastructure/pigeons/pair_provider.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:cobble/infrastructure/pigeons/pigeons.dart' as pigeon;
+import 'package:hooks_riverpod/all.dart';
+
+class PairCallbacks extends StateNotifier<int> implements pigeon.PairCallbacks {
```

Comment:
Nitpick, can you move out of pigeons folder? This folder only contains pigeon generated classes (my hope is that we could generate multiple files instead of having one huge file in the future, which is why I reserved whole folder for them).

Reply:
Done